tasks/fr/fr_income.py

```diff
@@ -0,0 +1,197 @@
+
+        subsectiontags = input_['subsections']
+        unittags = input_['unittags']
+        france = input_['sections']['fr']
+        insee_source = input_['sourcetag']['insee']
+
+        filepath = "incomemetadata/Income Variables - {}.tsv".format(self.table_theme)
+        with open(os.path.join(os.path.dirname(__file__),filepath)) as tsvfile:
+            tsvreader = csv.reader(tsvfile, delimiter="\t")
+            # Skip first row (header)
+            next(tsvreader, None)
+            for line in tsvreader:
+                var_code, short_name, long_name, universe, var_unit, \
+                    denominators, subsections = line
+                denominators = denominators.split(',')
+                targets_dict = {}
+                for x in denominators:
+                    x = x.strip()
+                    targets_dict[cols.get(x)] = 'denominator'
+                targets_dict.pop(None, None)
+                universes = universe.split(',')
+                for x in universes:
+                    x = x.strip()
+                    targets_dict[cols.get(x)] = 'universe'
+                targets_dict.pop(None, None)
+                cols[var_code] = OBSColumn(
+                    id=var_code,
+                    type='Numeric',
+                    name=long_name,
+                    description =long_name,
+                    # Ranking of importance, sometimes used to favor certain measures in auto-selection
+                    # Weight of 0 will hide this column from the user.  We generally use between 0 and 10
+                    weight=5,
+                    aggregate='',
+                    # Tags are our way of noting aspects of this measure like its unit, the country
+                    # it's relevant to, and which section(s) of the catalog it should appear in
+                    tags=[france, unittags[var_unit]],
+                    targets= targets_dict
+                )
+                subsections = subsections.split(',')
+                for s in subsections:
+                    s = s.strip()
+                    subsection_tag = subsectiontags[s]
+                    cols[var_code].tags.append(subsection_tag)
+                if var_unit in ('households', 'people', 'tax_consumption_units'):
+                    cols[var_code].aggregate = 'sum'
+
+        for _,col in cols.iteritems():
+            col.tags.append(insee_source)
+        return cols
+
+class FranceIncome(TableTask):
+
+    table_theme = Parameter()
+
+    def version(self):
+        return 2
+
+    def timespan(self):
+        return '2012'
+
+    def requires(self):
+        requirements = {
+            'data': RawIncomeIrisData(table_theme=self.table_theme),
+            'meta': IrisIncomeColumns(table_theme=self.table_theme),
+            'geometa': OutputAreaColumns(),
+        }
+        return requirements
+
+    def columns(self):
+        cols = OrderedDict()
+        cols['IRIS'] = self.input()['geometa']['dcomiris']
```

Comment:
this shouldn't be named `dcomiris` I see in other columns task they keep the name